deploy/cr/minimal/cr_minimal.yaml

```diff
@@ -3,4 +3,5 @@ kind: Infinispan
 metadata:
   name: example-infinispan
 spec:
+  image: infinispan/server:latest
```

Comment:
`image` is optional, so no longer a minimal example 😉. What's the reason for this change?